### DISCUSSION TITLE
Terminal session instead of single commands

### DISCUSSION THREAD

**Discussion by aapjeisbaas** [koreader/koreader#7605] [Ideas]
In the plugins I found a terminal plugin but this runs your input as `sh -c $INPUT ` and then reads its results back to you.
I would love to use my kobo as a small ssh terminal to connect to other machines or run a local tty.
Is this interactive tty mode something that would be possible to create with the koreader plugin system?

sidenote: I found a kindle project that supports local sessions using a gtk terminal emulator:
http://www.fabiszewski.net/kindle-terminal/

If anyone has another way to start a tty on a kobo plz share it ;-)

**Comment by Frenzie** [comment]
There's a "Konsole" in vlasovsoftlauncher that is interactive.

**Comment by NiLuJe** [comment]
https://github.com/llandsmeer/inkvt
